aten/src/ATen/native/cuda/SortingRadixSelect.cuh

```diff
@@ -121,7 +121,7 @@ struct TopKTypeConfig<at::Half> {
   typedef uint32_t RadixType;
 
   static inline __device__ RadixType convert(at::Half v) {
-#if CUDA_VERSION >= 8000 || defined __HIP_PLATFORM_HCC__
+#if CUDA_VERSION >= 9000 || defined __HIP_PLATFORM_HCC__
```

Comment:
Shouldn't you just delete this half of the condition now?

Reply:
If I delete the half, the following won't be enabled when compiling with __CUDACC__?
```
RadixType x = __half_as_ushort(v);
RadixType mask = -((x >> 15)) | 0x8000;
return (v == v) ? (x ^ mask) : 0xffff;
```

Reply:
Oh, I guess it should just be a test if you're compiling with `__CUDACC__`. I guess this is fine (the nit I have is that it implies that this code only works with CUDA 9.0, and that's not really true.)

Reply:
changed to `__CUDA_ARCH__` since that is guaranteed to be defined in device code.